pymilvus/milvus_client/milvus_client.py

```diff
@@ -0,0 +1,801 @@
+"""MilvusClient for dealing with simple workflows."""
+import logging
+from pprint import pprint
+import threading
+from typing import Optional, Union, List, Dict
+from uuid import uuid4
+
+from tqdm import tqdm
+
+from pymilvus.client.types import LoadState
+from pymilvus.exceptions import DataNotMatchException, MilvusException
+from pymilvus.milvus_client.defaults import DEFAULT_SEARCH_PARAMS
+from pymilvus.orm import utility
+from pymilvus.orm.collection import Collection, CollectionSchema, FieldSchema
+from pymilvus.orm.connections import connections
+from pymilvus.orm.types import DataType, infer_dtype_bydata
+
+logger = logging.getLogger()
+logger.setLevel(logging.DEBUG)
+
+
+class MilvusClient:
+    """The Milvus Client"""
+    def __init__(
+        self,
+        collection_name: str = "ClientCollection",
+        pk_field: str = None,
+        vector_field: str = None,
+        uri: str = None,
+        shard_num: int = 2,
+        partitions: List[str] = None,
+        consistency_level: str = "Session",
+        replica_number: int = 1,
+        index_params: dict = None,
+        timeout: Optional[int] = None,
+        drop_old: bool = False,
```

Comment:
drop may not be a good idea on init, especially on multi client situation.
I would rather let the user to drop the collection on close if they'd like to

Reply:
On close do you mean have a init arg drop_on_close and have the __delete__ also drop the collection? Or do we want dropping to only be called when the user calls it specifically. The reason I put this in so for the use case where you are doing a lot of testing and need a fresh collection each time.